build/scripts/docker/docker-compose-cc.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   cc:
-    image: "centrifugeio/centrifuge-chain:latest"
+    image: "centrifugeio/centrifuge-chain:20200207150243-0556d27"
```

Comment:
moved this since the latest cent chain needs metadata v11 but gsrcp doesn't support it yet